sw/device/lib/runtime/ibex.h

```diff
@@ -18,10 +18,37 @@
 
 /**
  * @file
- * @brief This header provides Ibex-specific functions, such as cycle-accurate
- * busy loops.
+ * @brief This header provides Ibex-specific functions and enums, such as
+ * cycle-accurate busy loops.
  */
 
+/**
+ * An Ibex exception type.
+ *
+ * This enum is used to decode RISC-V exception causes generated by Ibex.
+ */
+typedef enum ibex_exc_id {
+  kInstrMisaligned = 0,
+  kInstrAccessFault = 1,
+  kIllegalInstrFault = 2,
+  kBreakpoint = 3,
+  kLoadAccessFault = 5,
+  kStoreAccessFault = 7,
+  kUserECall = 8,
+  kMachineECall = 11,
+  kIdMax = 31
```

Comment:
nit: `k<type_name><variant_name>`. You may want to consider a shorter type name, e.g. `ibex_exc_t` and `kIbexExcInstrAccessFault`.

Reply:
fixed.